### ISSUE TITLE
[Accessibility] Check for alt in image tags

### ISSUE THREAD

**Issue by anonymous-sidhax** [anonymous-sidhax/Gryphon_Web_Crawler#14]

**Comment by anonymous-sidhax** [comment]
Suggestion:
alt_text_violations = []
    for img in soup.find_all('img'):
        if not img.find('alt'):
            abs_img_url = urljoin(url,img['src'])
            alt_text_violations.append((img,abs_img_url))

**Comment by anonymous-sidhax** [comment]
Done. Commit [#33](https://github.com/anonymous-sidhax/Gryphon_Web_Crawler/commit/0440a6675688101100f8bffbeade0bdceacd1b7e)
